client/routes/index.jsx

```diff
@@ -27,7 +27,7 @@ const NewScenario = props => {
 
 const Navigation = () => {
     return (
-        <Menu className="nav">
+        <Menu className="nav" stackable>
```

Comment:
I don't think we want this change in this PR?

Reply:
No, that was a mistake. I will remove

Reply:
Removed!